ArmTemplate/template.json

```diff
@@ -543,1266 +755,234 @@
-            "apiVersion": "2015-03-20",
-            "name": "[concat(parameters('workspaces_reliable_edge_relay_log_analytics_name'), '/LogManagement(', parameters('workspaces_reliable_edge_relay_log_analytics_name'), ')_General|dataPointsPerManagementGroup')]",
-            "dependsOn": [
-                "[resourceId('Microsoft.OperationalInsights/workspaces', parameters('workspaces_reliable_edge_relay_log_analytics_name'))]"
-            ],
-            "properties": {
-                "DisplayName": "Which Management Group is generating the most data points?",
-                "Category": "General Exploration",
-                "Query": "search * | summarize AggregatedValue = count() by ManagementGroupName\r\n// Oql: * | Measure count() by ManagementGroupName // Args: {OQ: True; WorkspaceId: 00000000-0000-0000-0000-000000000000} // Settings: {PTT: True; SortI: True; SortF: True} // Version: 0.1.122",
-                "Version": 2
-            }
-        },
-        {
-            "type": "Microsoft.OperationalInsights/workspaces/savedSearches",
-            "apiVersion": "2015-03-20",
-            "name": "[concat(parameters('workspaces_reliable_edge_relay_log_analytics_name'), '/LogManagement(', parameters('workspaces_reliable_edge_relay_log_analytics_name'), ')_General|dataTypeDistribution')]",
+            "type": "Microsoft.Storage/storageAccounts/blobServices",
+            "apiVersion": "2019-04-01",
+            "name": "[concat(parameters('storageAccounts_reliableedgerelaydiag_name'), '/default')]",
             "dependsOn": [
-                "[resourceId('Microsoft.OperationalInsights/workspaces', parameters('workspaces_reliable_edge_relay_log_analytics_name'))]"
+                "[resourceId('Microsoft.Storage/storageAccounts', parameters('storageAccounts_reliableedgerelaydiag_name'))]"
             ],
             "properties": {
-                "DisplayName": "Distribution of data Types",
-                "Category": "General Exploration",
-                "Query": "search * | extend Type = $table | summarize AggregatedValue = count() by Type\r\n// Oql: * | Measure count() by Type // Args: {OQ: True; WorkspaceId: 00000000-0000-0000-0000-000000000000} // Settings: {PTT: True; SortI: True; SortF: True} // Version: 0.1.122",
-                "Version": 2
+                "cors": {
+                    "corsRules": []
+                },
+                "deleteRetentionPolicy": {
+                    "enabled": false
+                }
             }
         },
         {
-            "type": "Microsoft.OperationalInsights/workspaces/savedSearches",
-            "apiVersion": "2015-03-20",
-            "name": "[concat(parameters('workspaces_reliable_edge_relay_log_analytics_name'), '/LogManagement(', parameters('workspaces_reliable_edge_relay_log_analytics_name'), ')_General|StaleComputers')]",
+            "type": "Microsoft.Storage/storageAccounts/blobServices",
+            "apiVersion": "2019-04-01",
```

Comment:
Api version in most of the cases is not the latest in this ARM template because of the impact regarding changed structure and different props.
Nevertheless...an update for blobServices is reasonable and has no affect on the structure here.